modules/local/cogeqc.nf

```diff
@@ -4,39 +4,33 @@ process COGEQC {
 
     container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/cogeqc-1.2.0_r-4.2.2':
         '' }"
-        
+
     publishDir(
         path: "${params.outdir}/orthogroup-summaries",
-        mode: 'copy',
+        mode: params.publish_dir_mode,
```

Comment:
Nextflow best practices thing. `publish_dir_mode` is a config variable already set to `copy`. If we wanted to overwrite that value we can do so here. But in Austin's case, he doesn't want to overwrite (not yet anyway), so I'd rather set the codebase such that I can change the value of `mode` in a single place rather than in all of the processes.

Reply:
love that, thank you for the explanation